lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -567,3 +574,45 @@ def test_free_course_display(self):
         self.assertEqual(resp.status_code, 200)
         self.assertNotIn("Add free to Cart (Free)", resp.content)
         self.assertNotIn('<p class="important-dates-item-title">Price</p>', resp.content)
+
+
+class CourseAboutTestCaseCCX(SharedModuleStoreTestCase, LoginEnrollmentTestCase):
+    """
+    Test for unenrolled student tries to access ccx.
+    Note: Only CCX coach can enroll a student in CCX. In sum self-registration not allowed.
+    """
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Set up course for tests
+        """
+        super(CourseAboutTestCaseCCX, cls).setUpClass()
+        cls.course = CourseFactory.create()
+
+    def setUp(self):
+        """
+        Set up tests
+        """
+        super(CourseAboutTestCaseCCX, self).setUp()
+
+        # Create ccx coach account
+        self.coach = coach = AdminFactory.create()
+        self.client.login(username=coach.username, password="test")
```

Comment:
@singingwolfboy  please see code below:

[AdminFactory](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/tests/factories.py#L114) is child class of 
[UserFactory](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/tests/factories.py#L71). And inside ``UserFactory`` we have [hard coded password test](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/tests/factories.py#L78)